docs/whats-new.rst

```diff
@@ -37,6 +37,12 @@ Breaking Changes
 - Deprecate ``func_input_dtype`` argument to ``Var`` (fixes
   :issue:`281` via :pull:`282`).  By `Spencer Hill
   <https://github.com/spencerahill>`_.
+- Improve compatibility for data following IRIDL conventions or NOAA
+  data formats. Specifically, several alternate names are defined in
+  `GRID_ATTRS`, while there is no longer an assumption that 
```

Comment:
In restructured text, which is what we use to write the documentation, [inline literals](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#inline-literals) are specified using double back-ticks, e.g.:
```
``GRID_ATTRS``
```
So add a couple more back-ticks here for `GRID_ATTRS` as well as some for `BOUNDS_STR` and `time_weights`.

This is slightly different than GitHub Markdown, so it takes some time getting used to...

Reply:
whoops! I'm on it. 
@spencerkclark is the procedure to close this PR, push another commit to my branch, and then submit a new PR?

Reply:
No need to close the PR!  Just push a new commit to the branch and it will show up here.

Reply:
You mean like where it says 

> "Add more commits by pushing to the noaa-iridl-compat branch on jdossgollin/aospy."

down below? 😄

Reply:
Perfect!